src/main/java/software/amazon/event/kafkaconnector/mapping/JsonMapper.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.mapping;
+
+import static java.util.Collections.singletonMap;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import java.io.IOException;
+import org.apache.kafka.connect.header.Header;
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+public class JsonMapper {
+  public final JsonConverter jsonConverter = new JsonConverter();
```

Comment:
We use the jsonConcerter in in our TimeMapper interface. It would be nice to have it available, as dicussed in #358. Will make the objectMapper private.

Reply:
To my understanding the purpose of extracting this behavior to the new class was to provide the new custom TimeMapper implementations a unified way to access the deserialized SinkRecord (what createJsonPayload does). If your custom implementation has to rely on this class internals such as the JsonConverter or the createHeaderArray function then we chose the wrong abstraction. 

Can you elaborate on why your custom time mapper implementation can not work with the result of createJsonPayload? It basically contains the results of the createJSONFromByteArray already. Please also let me know where you need the JSON Converter.

cc @embano1